dir.targets

```diff
@@ -93,4 +93,76 @@
     </ItemGroup>
   </Target>
 
+  <!-- Support for CoreRT Targeting Packs
+       CoreRT Targeting requires differentiating the targetting pack according to the platforms.
+       i.e.: the targetting pack for linux is different than that of windows. In order to support this
```

Comment:
Looks like there was a misunderstanding. I thought @jkotas were stating that due to differences in System.Private.Interop we would need different targeting pack for each platform. And that's why I designed the production and consumption of the targeting packs of CoreRT in this way. Apparently, the differences in System.Private.Interop won't necessarily be effective in consumption; we might not need to reference/consume those platform specific APIs  from CoreFX libraries, in which case using Windows x64 assets for all platforms would be fine. 

I'll close this PR and will prep another one according to that plan.